src/applications/vaos/containers/TypeOfCarePage.jsx

```diff
@@ -40,6 +41,7 @@ const pageKey = 'typeOfCare';
 export class TypeOfCarePage extends React.Component {
   componentDidMount() {
     this.props.openFormPage(pageKey, uiSchema, initialSchema);
+    this.props.getUserSystems();
```

Comment:
Fetches systems on load of Type Of Care page so that we have them available in `next()`

Reply:
This is just to make things slightly faster, correct? You could do the fetch in `next()`.